src/foremast/configs/outputs.py

```diff
@@ -86,15 +86,20 @@ def write_variables(app_configs=None, out_file='', git_short=''):
         if env != 'pipeline':
             instance_profile = generated.iam()['profile']
             rendered_configs = json.loads(
-                get_template('configs/configs.json.j2', env=env, app=generated.app_name(), profile=instance_profile))
+                get_template(
+                    'configs/configs.json.j2',
+                    env=env,
+                    app=generated.app_name(),
+                    profile=instance_profile,
+                    generated=generated))
```

Comment:
Can we make this more clear to what it is? `generated_data` or `templated_data` or something along those lines? I think it will make it easier long term. Feel free to discuss.

Reply:
I think that is a valid point. I just wanted to have something out there for feedback.
How about `formats`?

It seems simple and right to the point and we know the dataset is coming from the `formats` configs: http://foremast.readthedocs.io/en/latest/configuration_files/foremast_config.html#formats.

Reply:
Keeps it consistent. If I search `formats`, ill see it came from `utils` which is good.